src/sentry/demo/demo_start.py

```diff
@@ -174,9 +174,21 @@ def get_one_release(org: Organization, project_slug: Optional[str]):
 
 def get_one_issue(org: Organization, project_slug: Optional[str]):
     group_query = Group.objects.filter(project__organization=org)
-    if project_slug:
+    if project_slug == "react-native":
+        group_query = group_query.filter(project__slug=project_slug)
+        for group in group_query:
+            if "Promise" in group.message:
```

Comment:
@roggenkemper what's up with these checks for `Promise` and `Bad`?

Reply:
that's to make sure it's the correct error that we send.

Reply:
since it is a EXC_BAD_ACCESS error and a promise argument issue

Reply:
@roggenkemper We want something more generic than this. What I'm asking for is the `demo_start` view to accept a new parameter called `errorType` then we find an issue that matches that type. If no error exists of that type, just return any error for that project

Reply:
I see. ill get on that now!